models/review.go

```diff
@@ -53,8 +53,9 @@ type Review struct {
 	IssueID    int64  `xorm:"index"`
 	Content    string `xorm:"TEXT"`
 	// Official is a review made by an assigned approver (counts towards approval)
-	Official bool `xorm:"NOT NULL DEFAULT false"`
-	Stale    bool `xorm:"NOT NULL DEFAULT false"`
+	Official bool   `xorm:"NOT NULL DEFAULT false"`
+	CommitID string `xorm:"VARCHAR(40)"`
```

Comment:
That's good to add this and I think we also need a migration to fill old columns. I think code comments have saved the commitid and both reviewid. So I think it can be done.

Reply:
I think it would be hard in any sane and reliable way to recreate review commit id in migration step. Code comments save the commit id from the line blame the comment is made on, which might be an earlier commit than the complete review is done on. Also a review can be done without any code comments.
It might be possible to use the _UpdatedUnix_ time of the review and find the commit which is closest in time before that. But I am not sure if that is a good idea.

Reply:
`git rev-list -1 --before="<updated time>" <pr-head-branch>`

But it is not completly reliable because commit timestamps can be spoofed and set to newer or older time.

Reply:
I think it would be ok to not this in migration as it would only affect PR currently in review